siriuspy/siriuspy/devices/blctrl.py

```diff
@@ -1,9 +1,46 @@
 """Beamline Control."""
```

Comment:
maybe we could rename this module to `ppsctrl`?

Reply:
done

Reply:
@xresende , it seems the file was deleted

Reply:
@it was renames to `intlkctrl.py` since now it has classes for machine MPS and PPS interlocks.